upnext/scrape.py

```diff
@@ -0,0 +1,48 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+import time
+from bs4 import BeautifulSoup
+from .models import Party, Track
+import spotipy
+import tokens
+
+
+def get_currently_playing(party, embed):
+  token_info = tokens.token_read()
+  sp = spotipy.Spotify(auth=token_info['ACCESS_TOKEN'])
+  # playlist_url = get_playlist_url()
+  party_id = party.uri.split(':')[-1]
+
+  profile = webdriver.FirefoxProfile()
+  profile.set_preference("webdriver.load.strategy", "unstable")
+  browser = webdriver.Firefox(firefox_profile=profile)
+  browser.get(embed)
+  time.sleep(3)
```

Comment:
then they shouldnt be streaming music LOL

but 3 was more than enough for us at least so it should be safe? we can make it more once it's a background process